CHANGES.rst

```diff
@@ -1,6 +1,33 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-19)
+--------------------------
+
+- Improves command line tool documentation by grouping of commands and
+  command examples.
+- Centralises the way workflow name is passed to CLI commands.
+- Renames ``workflows`` to ``list`` and ``list`` to ``ls``.
+- Introduces ``du`` command for checking workspace disk usage.
+- Introduces ``mv`` command for moving files within workspace.
+- Introduces ``diff`` command for comparing two workflows.
+- Introduces ``rm`` command for removing files within workspace.
+- Introduces ``stop`` command for stopping a running workflow.
+- Introduces ``delete`` command for deleting a workflow.
+- Introduces ``run`` command which wraps ``create``, ``upload`` and ``start``
+  commands.
+- Introduces commands for opening, listing and closing interactive sessions.
+- Introduces ``--parameter`` flag for overriding workflow input parameters.
+- Introduces ``--option`` flag for passing operational options to ``start`` or
```

Comment:
Maybe here we should add a small example of what an operational option would consist of, I feel it is too abstract.

We do it in help messages, see:
```console
$ reana-client run --help
...
  -o, --option TEXT        Additional operatioal options for the workflow
                           execution. E.g. CACHE=off.
```

BTW, I just spotted a typo in the help message now, it is [here](https://github.com/reanahub/reana-client/blob/master/reana_client/cli/workflow.py#L674).